goreader.go

```diff
@@ -41,14 +34,15 @@ func main() {
 	defer rc.Close()
 	book := rc.Rootfiles[0]
 
-	a := app{book: book}
+	a := app{book: book, exitSignal: make(chan bool, 1)}
 	a.run()
 
-	if a.err == nil || a.err == exitRequest {
-		os.Exit(0)
-	} else {
+	if a.err != nil {
+		fmt.Fprintf(os.Stderr, "Exit with error: %s\n", err.Error())
```

Comment:
Did you mean to use `a.err.Error()` here?

Reply:
Totally! Gonna fix it asap. :wink:

Reply:
Fixed in 1b74a35.